includes/ucf-social-common.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Place common functions here.
+ **/
+
+if ( ! class_exists( 'UCF_Social_Common' ) ) {
+	class UCF_Social_Common {
+
+		/**
+		* Displays the social social icons
+		* @author RJ Bruneel
+		* @since 1.0
+		* @param array $atts | Assoc. array of shortcode options
+		* @return string
+		**/
+		public static function display_social_icons( $atts ) {
+				$before = self::ucf_social_icons_display_before();
```

Comment:
You should pass $atts into the before and after hooks even if you don't use them. If someone wants to override the layouts, they may want to pass something through the $atts variable to conditionally lay something out.